### PR TITLE
Add bundler-audit travis job

### PR DESCRIPTION
**Pull request by dirkmueller**

### CODE REVIEW

**Comment by aspiers** [comment]
Nice idea!

File: crowbar_framework/Gemfile [hunk 1]
```diff
@@ -67,6 +67,7 @@ group :test do
 
   gem "cucumber-rails", "~> 1.4.2", require: false
   gem "simplecov", "~> 0.9.1", require: false
+  gem "bundler-audit", "~> 0.4.0"
```
Comment (tboerger) [inline crowbar_framework/Gemfile]: This block are just ```reuire: false``` rubygems, just move it to the upper block of rubygems.

File: .travis.yml [hunk 2]
```diff
@@ -1,4 +1,5 @@
 language: ruby
+sudo: false
```
Comment (tboerger) [inline .travis.yml]: This should be a separate commit ;)

Comment (dirkmueller) [inline .travis.yml]: It is now, thanks

**Comment by toabctl** [comment]
+1

**Comment by tboerger** [comment]
:+1:
